docs/01-overview/main-areas/application-connectivity/README.md

```diff
@@ -5,11 +5,15 @@ title: What is Application Connectivity in Kyma?
 Application Connectivity in Kyma is an area that: 
 
 - Simplifies and secures the connection between external systems and Kyma
+<!-- TODO: is this one still true?
 - Registers external events and APIs and simplifies the API usage
+-->
+<!-- TODO: is this one still true?
 - Provides asynchronous communication with services and Functions deployed in Kyma through events
+-->
 - Manages secure access to external systems
 - Provides monitoring and tracing capabilities to facilitate operational aspects
 
 Depending on your use case, Application Connectivity works in one of the two modes: 
-- **Legacy mode** (default) - using components such as [Application Registry](ac-03-application-registry.md) and [Connector Service](../../../05-technical-reference/00-architecture/ac-02-connector-service.md)
+- **Legacy mode** (default) - a standalone mode where Kyma is not connected to Compass
 - **Compass mode** - using [Runtime Agent](ra-01-runtime-agent-overview.md) and integration with [Compass](https://github.com/kyma-incubator/compass)
```

Comment:
We can extend this description:
```suggestion
- **Compass mode** - using [Runtime Agent](ra-01-runtime-agent-overview.md) and integration with [Compass](https://github.com/kyma-incubator/compass) to automate connection and registration of Services using mTLS certificates.
```

Reply:
And by `Services` here you intentionally mean Kubernetes Services? (The ones we register and secure and capitalize in our docs) Just double checking.

Reply:
I mean external services that are registered by the external systems (i.e. rest APIs or other things registered by those)